### DISCUSSION TITLE
[Bug] In ExplorerPatcher's properties, in the "System Tray" category, it is possible to enable nonexistent/broken options in Windows 10

### DISCUSSION THREAD

**Discussion by sdw4938** [valinet/ExplorerPatcher#1006] [General]
In Windows 10 in the System Tray category, when changing the network flyout there is a "Control Center" option which doesn't do anything since the control center is absent in Win10. Same with the Calendar flyout, there is an option to make the date and time button to open the action center which works, however, there isn't a calendar in the action center in Windows 10.
![Screenshot 2022-03-05 212712](https://user-images.githubusercontent.com/76626288/156906558-c4845600-83d7-4900-8614-31d31aa328c4.png)
![Screenshot 2022-03-05 212654](https://user-images.githubusercontent.com/76626288/156906559-7a282d67-8423-4d5f-9166-82201ed886a1.png)

**Comment by valinet** [comment]
1. "Control Center" there doesn't make sense, will remove it in Windows 10 indeed
2. "Action Center", even though not having a calendar, if it works, I will leave it in there, maybe some people prefer that still, for some reason. Since it's there for free, I don't mind it.
